back/engines/commercial/analytics/db/views/analytics_fact_participations_v06.sql

```diff
@@ -0,0 +1,163 @@
+FROM reactions r
+INNER JOIN analytics_dimension_types adt ON adt.name = 'reaction' AND adt.parent = LOWER(r.reactable_type)
+LEFT JOIN ideas i ON i.id = r.reactable_id
+LEFT JOIN comments c ON c.id = r.reactable_id
+LEFT JOIN ideas ic ON ic.id = c.post_id
+
+UNION ALL
+
+-- Poll Response
+SELECT
+    pr.id,
+    pr.user_id AS dimension_user_id,
+    COALESCE(pr.user_id::CHAR, pr.id::CHAR) as participant_id,
+    COALESCE(p.project_id, pr.phase_id) AS dimension_project_id,
+    adt.id AS dimension_type_id,
+    pr.created_at::DATE AS dimension_date_created_id,
+    0 AS reactions_count,
+    0 AS likes_count,
+    0 AS dislikes_count
+FROM polls_responses pr
+LEFT JOIN phases p ON p.id = pr.phase_id
+INNER JOIN analytics_dimension_types adt ON adt.name = 'poll'
+
+UNION ALL
+
+-- Volunteering
+SELECT
+    vv.id,
+    vv.user_id AS dimension_user_id,
+    COALESCE(vv.user_id::CHAR, vv.id::CHAR) as participant_id,
+    COALESCE(p.project_id, vc.phase_id) AS dimension_project_id,
+    adt.id AS dimension_type_id,
+    vv.created_at::DATE AS dimension_date_created_id,
+    0 AS reactions_count,
+    0 AS likes_count,
+    0 AS dislikes_count
+FROM volunteering_volunteers vv
+LEFT JOIN volunteering_causes vc ON vc.id = vv.cause_id
+LEFT JOIN phases p ON p.id = vc.phase_id
+INNER JOIN analytics_dimension_types adt ON adt.name = 'volunteer'
+
+UNION ALL
+
+--Baskets
+SELECT
+    b.id,
+    b.user_id AS dimension_user_id,
+    COALESCE(b.user_id::CHAR, b.id::CHAR) as participant_id,
+    p.project_id AS dimension_project_id,
+    adt.id AS dimension_type_id,
+    b.created_at::DATE AS dimension_date_created_id,
+    0 AS reactions_count,
+    0 AS likes_count,
+    0 AS dislikes_count
+FROM baskets b
+LEFT JOIN phases p ON p.id = b.phase_id
+INNER JOIN analytics_dimension_types adt ON adt.name = 'basket'
+
+UNION ALL
+
+--Event attendance
+SELECT
+    ea.id,
+    ea.attendee_id AS dimension_user_id,
+    COALESCE(ea.attendee_id::CHAR, ea.id::CHAR) as participant_id,
+    e.project_id AS dimension_project_id,
+    adt.id AS dimension_type_id,
+    ea.created_at::DATE AS dimension_date_created_id,
+    0 AS reactions_count,
+    0 AS likes_count,
+    0 AS dislikes_count
+FROM events_attendances ea
+LEFT JOIN events e ON e.id = ea.event_id
+INNER JOIN analytics_dimension_types adt on adt.name = 'event_attendance'
+
+UNION ALL
+
+--Followers
+SELECT
+  f.id,
+  f.user_id AS dimension_user_id,
+  COALESCE(f.user_id::CHAR, f.id::CHAR) as participant_id,
+  CASE f.followable_type
+    WHEN 'Project' THEN f.followable_id
+    WHEN 'Idea' THEN i.project_id
+    ELSE NULL
+  END AS dimension_project_id,
+  adt.id AS dimension_type_id,
+  f.created_at::DATE AS dimension_date_created_id,
+  0 AS reactions_count,
+  0 AS likes_count,
+  0 AS dislikes_count
+FROM followers f
+INNER JOIN analytics_dimension_types adt ON adt.name = 'follower' AND adt.parent = LOWER(f.followable_type)
+LEFT JOIN ideas i ON i.id = f.followable_id;
```

Comment:
Missing new line at the end of the file.
```suggestion
LEFT JOIN ideas i ON i.id = f.followable_id;

```